src/pages/index.js

```diff
@@ -6,7 +6,7 @@ import '../components/youtube-playlist/youtube-playlist.js';
 
 export default class Home extends HTMLElement {
   async connectedCallback() {
-    const events = (await fetch('https://www.analogstudios.net/api/v2/events/?tag=tt')
+    const events = (await fetch('https://www.analogstudios.net/api/v2/events?tag=tt')
```

Comment:
Seems related to changes made as part of https://github.com/AnalogStudiosRI/www.analogstudios.net/issues/81